mlrun/db/httpdb.py

```diff
@@ -3264,6 +3266,41 @@ def _wait_for_background_task_to_end(bg_name):
 
         return bg_task
 
+    def get_datastore_profile(
+        self, name: str, project: str
+    ) -> Optional[mlrun.common.schemas.DatastoreProfile]:
+        project = project or config.default_project
+        path = self._path_of("projects", project, "datastore_profiles") + f"/{name}"
+
+        res = self.api_call(method="GET", path=path)
+        if res and res._content:
+            public_wrapper = json.loads(res._content)
+            datastore = DatastoreProfile2Json.create_from_json(
+                public_json=public_wrapper["body"]
+            )
+            return datastore
+        return None
+
+    def delete_datastore_profile(self, name: str, project: str):
+        pass
+
+    def list_datastore_profile(
```

Comment:
`list_datastore_profile` -> `list_datastore_profiles` (plural).
Also, why not implemented?

Reply:
wanted to expose it externally in a separate PR